### PR TITLE
(1) readme udpated, (2) CCUTIL bug fixed and (3)Minor updates to example

### PR DESCRIPTION
**Pull request by jstaffor**
(1) readme udpated
(2) update to repo based on a CCUTIL bug fixed I encountered
(3) Very minor updates to example in repo
(4) ".svg" image moved to "images" folder

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -96,92 +34,88 @@ In addition, this repository structure has been designed to work with the follow
...
-  * You must set the Content Directory for each book to <docs>/<title>
-  * 
+The Chapters are found within the Book (*master.adoc*).
+They are simply headings.
+Each Chapter will contain one to many User Stories.
 
-//// 
-Fix buildGuides.sh to work with multi-topic-level and re-include this info
+In *master.adoc*, the User Stories are formed using references to Modular Doc *.adoc* files.
+The structure of the User Story is defined with headings and the structure is based on an link:https://gitlab.cee.redhat.com/ccs-internal-documentation/Modular_Documentation_Project/tree/master/files[Assembly template].
```
Comment (wtrocki) [inline README.adoc:68]: Internal link in upstream.  If that's something important we should copy it to upstream as well.

File: README.adoc [hunk 2]
```diff
@@ -96,92 +34,88 @@ In addition, this repository structure has been designed to work with the follow
...
+A User Story is created using Modular Docs: link:https://gitlab.cee.redhat.com/ccs-internal-documentation/Modular_Documentation_Project/tree/master/files[Concept, Procedure and Reference Templates].
 
+To best understand the information above, see the example located in the repo.
+
+== Building the Community Edition
+
+The *master.adoc* file is the only file to build when rendering the files into text.
+Build the example located in the _upstream-1_ folder:
+
+    gem install asciidoctor
```
Comment (wtrocki) [inline README.adoc:113]: ping @austincunningham - We need to replace this with scripting.

**Review by wtrocki — APPROVED** [review]
Tech approved. 
Minor comments added.

File: README.adoc (hunk 1 above)
Comment (jstaffor) [reply README.adoc:68]: They are the template that people can use - they are a great guide and so I would consider them important. I'll figure out where best to put them in the repo and create another PR.

**Pull request merged**